### DISCUSSION TITLE
Why is lato missing some font weights?

### DISCUSSION THREAD

**Discussion by techopswhiz** [fontsource/fontsource#790] [Q&A]
For example font weight 500 doesn't seem to exist, but this limitation does not exist if you were to use google fonts.

**Comment by wasit-shafi** [comment]
@menasheh I guess you may not have integrated the custom fonts properly, to ensure that all the font weights works, you may need more than one font family asset eg. lato-hairline.woff2, lato-medium.woff2, lato-bold.woff2, lato-regular.woff2 etc under the common font-family name, the multiple font assets act as a fallback to ensure all the font weights work correctly. If I am not wrong the only reason why Google font was working in your case may be because the CDN URL you are using are passing metadata to include that font weights say  this link `<link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;1,300&display=swap" rel="stylesheet">`  clearly mentions to include font weights of 100, 300, 400, below is the code snippet to add custom font family by providing own font asset file
```
@font-face {
	font-family: lato;
	src: url('../assets/fonts/lato-thin.woff2') format('woff2');
	font-style: normal; // for italic font asset use font-style: italic;
	font-weight: 250;
}
// Fallbacks below
@font-face {
	font-family: lato;
	src: url('../assets/fonts/lato-light.woff2') format('woff2');
	font-style: normal;
	font-weight: 300;
}

@font-face {
	font-family: lato;
	src: url('../assets/fonts/lato-regular.woff2') format('woff2');
	font-style: normal;
	font-weight: 400;
}

@font-face {
	font-family: lato;
	src: url('../assets/fonts/lato-medium.woff2') format('woff2');
	font-style: normal;
	font-weight: 500;
}

@font-face {
	font-family: lato;
	src: url('../assets/fonts/lato-semi-bold.woff2') format('woff2');
	font-style: normal;
	font-weight: 600;
}

@font-face {
	font-family: lato;
	src: url('../assets/fonts/lato-bold.woff2') format('woff2');
	font-style: normal;
	font-weight: 700;
}
```
Thanks
